src/components/ViewComments.jsx

```diff
@@ -1,26 +1,90 @@
-const ViewComments = ({ isLoadingComments, isReadingComments, comments }) => {
+import { useEffect, useState } from "react";
+import { postCommentToReview } from "../api";
+import { getCommentsOnReview } from "../api";
+
+const ViewComments = ({ isReadingComments, review_id }) => {
+  const [comment, setComment] = useState("");
+  const [comments, setComments] = useState([]);
+  const [isLoadingComments, setIsLoadingComments] = useState(false);
+  const [postError, setPostError] = useState(false);
+
+  useEffect(() => {
+    setIsLoadingComments(true);
+    getCommentsOnReview(review_id).then((data) => {
+      setComments(data.reverse());
+      setIsLoadingComments(false);
+    });
+  }, []);
+
+  const handleSubmit = (event, review_id) => {
+    event.preventDefault();
+    postCommentToReview(review_id, comment)
+      .then(() => {
+        getCommentsOnReview(review_id).then((data) => {
```

Comment:
Rarther than running another API call to fetch the comments again you could append the new comment to the start or end of the current comments array optimisticaly and use the return comment from your postComment function to replace this temporary comment with the actual comment once this is returned from the API to save the extra API call.